types/ckeditor__ckeditor5-utils/src/difftochanges.d.ts

```diff
@@ -1,4 +1,4 @@
-import { Change } from "@ckeditor/ckeditor5-utils/src/diff";
+type Change = "equal" | "insert" | "delete";
```

Comment:
This is a perfect example of why `Change` should be exported, above. There's no need for the repetition.

Reply:
@rbuckton If you don't mind that I export a type (or interface) that is not marked as exported in the official documentation, I can make those modifications. :)